tfhe/src/high_level_api/zk.rs

```diff
@@ -1,13 +1,26 @@
 use crate::zk::CompactPkeCrs;
-use crate::Config;
+use crate::{Config, Error};
 
 impl CompactPkeCrs {
     /// Create a new `CompactPkeCrs` from a `Config` object.
     /// max_bit_size is the maximum number of bits that can be proven, e.g. 64 for a single
     /// FheUint64 or 8 x FheUint8 values.
+    ///
+    /// This function assumes that packing will be applied during ZK proof.
     pub fn from_config(config: Config, max_bit_size: usize) -> crate::Result<Self> {
```

Comment:
Probably not so easy, but it might be worth to explain that since we assume packing, the actual number of bits proved bits is `max_bit_size * 2` 🤔

Reply:
no it's still max_bit_size, but proven over k/2 slot if k is the number of unpacked slots

Reply:
ah yes ok